### DISCUSSION TITLE
Trouble Running Mastodon With Docker-Compose

### DISCUSSION THREAD

**Discussion by stawze** [mastodon/mastodon#24071] [General]
Hi, I'm having issues running Mastodon with Docker.

I get the following error on the mastodon_web_1 container:

/opt/mastodon/vendor/bundle/ruby/3.0.0/gems/activerecord-6.1.7.2/lib/active_record/connection_adapters/postgresql/database_statements.rb:19:in `exec': ERROR:  relation "accounts" does not exist (PG::UndefinedTable)
LINE 8:  WHERE a.attrelid = '"accounts"'::regclass

What am I doing wrong?

**Comment by DejavuMoe** [comment]
It looks like a database issue, did you do a database initialization setup before you first time started the Mastodon container? (It will pre-set the database)

**Reply by nickjuntilla** [reply]
Is there a step by step guide to running this with docker anywhere? I just see the Docker and docker-compose file, but when I build the image and run I get tons of errors. I think we're missing a lot of information.
